content/momentum/4/4-tls-macros.md

```diff
@@ -15,71 +15,18 @@ The tls_macros module must be loaded in a configuration file, as follows:
 `tls_macros {}`
 ### <a name="tls_macros.macros"></a> TLS Logging Macros
 
-[Table 71.2, “TLS Logging Macros”](/momentum/4/4-tls-macros#tls-macros-table) lists the TLS logging macros.
-
 <a name="tls-macros-table"></a> 
 
-
 | Macro | Description | Examples |
 | --- | --- | --- |
-| outbound_tls:outbound_tls | Whether TLS was used or not | 
```

Comment:
It appears that none of these macros is only applicable to TLS1.2 or below.  All the changes were because the table syntax was fubar and made for a rather ugly page.